Library/Formula/open-ocd.rb

```diff
@@ -2,13 +2,14 @@
 
 class OpenOcd < Formula
   homepage 'http://sourceforge.net/projects/openocd/'
-  url 'https://downloads.sourceforge.net/project/openocd/openocd/0.8.0/openocd-0.8.0.tar.bz2'
-  sha1 '10bf9eeb54e03083cb1a101785b2d69fbdf18f31'
-
-  devel do
-    url 'https://downloads.sourceforge.net/project/openocd/openocd/0.9.0-rc1/openocd-0.9.0-rc1.tar.bz2'
-    sha1 '8ff9dca7751f7f4becca1507fd74df4405c5155e'
-  end
+  url 'https://downloads.sourceforge.net/project/openocd/openocd/0.9.0/openocd-0.9.0.tar.bz2'
+  sha1 'bcf99bb21f09513065b2c9ece148ace7d16cdd0c'
+
+  # There is no devel version currently, just latest release and git master
```

Comment:
If there's no `devel` version let's cut the entire block. We can re-add it when there is one.